_posts/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "OpenSearch Partner Highlight: Using Cloaked Search to Protect Your Data"
+authors:
+  - zmre
+date:   2022-11-09 1
+categories:
+  - partners
+---
+
+Most companies will tell you that the data they hold is well protected. They'll say it's encrypted “at-rest and
+in-transit,” which means they use HTTPS and disk-level or database-level encryption. Unfortunately, this isn't
+the protection most people think it is.
+
+## Most Encryption Claims Are Smoke and Mirrors
+
+What passes for data protection these days is often wildly overblown on the “protection” front.
+
+HTTPS protects the data for a brief moment in time by warding off eavesdroppers and interlopers, but it
+**doesn't change the availability of the data**. For example, this blog is served over HTTPS, and you are
+probably not authenticated.
+
+“At-rest encryption” is really only useful when someone steals a hard drive. When used by an always-on server,
+the value drops near to zero. That's because the encryption key is only needed when the system starts up and
+after that, the protection is transparent to anybody accessing the server.
+
+![Application Layer Encryption]({{ site.baseurl }}/assets/media/blog-images/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search/app-layer-enc.png){:class="img-centered"}
+
+## Application-layer Encryption: A Better Approach
+
+When you encrypt the data before it is sent to the storage layer, an attacker scraping a database gets a bunch
+of garbage bytes that aren't useful to them without the relevant keys. This is known as application-layer
+encryption (ALE) because it happens at the layer above the data store instead of below it.
+
+By separating the data and keys, companies can achieve meaningful defense-in-depth where the everyday problems
+of cloud misconfigurations, application vulnerabilities, and network breaches don't necessarily result in a
+data breach. This makes gaining access to the needed keys an entirely different and more difficult problem for
+an attacker.  
+
+If this approach were used more often, we wouldn't constantly have stories about attackers that get into a
+corporate network, move laterally within the network to a database server, and then scrape all of the data
+down for sale on the darknet. The companies who are implementing this correctly are the ones who aren't in
+the news.
+
+## Why Encrypt Data in OpenSearch?
+
+Search services are often an underprotected backdoor way to gain access to sensitive data. They frequently hold
+personal information like names and addresses. They can hold log data that includes web form submissions. They
+may hold communications records like emails, customer messages, and chats. They may also hold insider-trading
+fodder like roadmap info, financial projections, legal documents, and much more.
```

Comment:
In this paragraph, you might consider combining a few of the sentences so that you avoid a staccato tone.